### DISCUSSION TITLE
How to access `AutoGluonTabularModel` for timeSeries model?

### DISCUSSION THREAD

**Discussion by guilhermeparreira** [autogluon/autogluon#3274] [Q&A] [answered]
Hi There! 

I am creating two dictionaries of models:

+ 1 that accepts features
+ Another that does not;

I am trying to use [source code](https://github.com/autogluon/autogluon/blob/d9e7beca5a9df9cbe8543f5750f02a47201403e8/timeseries/src/autogluon/timeseries/models/presets.py), [model-zoo](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-model-zoo.html#module-autogluon.timeseries.models) and the [list of models that accept features](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-model-zoo.html#additional-features) to map the correct names. What I got so far is:

```
model_with_covariates = {
    'DeepAR': {},
    'TemporalFusionTransformer': {},
 }

model_without_covariates = {
    'Naive': {},
    'SeasonalNaive': {},
    'ARIMA': {},
    'ETS': {},
    'Theta': {},
    'AutoETS': {},
    'AutoARIMA': {},
    'DynamicOptimizedTheta': {},
    'SimpleFeedForward': {},
}
```

But I didn't find the alias for `AutoGluonTabularModel` . 

Context: I am doing TSCV and want to know the impact of features in prediction. Separating these models saves time (a model that can't run with a feature won't be requested.

TLDR

What is the correct name to add  `AutoGluonTabularModel` to my dictionary? (I want to try a 'regression' fashion for this time series problem).

**Comment by shchur** [comment, marked as answer]
Hi, you should be able to access the model as `AutoGluonTabular`.

As a side note, I would recommend to check out the pre-release version of AutoGluon, where we refactored & significantly improved the tabular-based models. You can install the new version with `pip install --pre autogluon`, and the new tabular models can be accessed as `RecursiveTabular` and `DirectTabular` (this used to be called `AutoGluonTabular` in the v0.7.0 release).

**Reply by guilhermeparreira** [reply]
I did `pipenv install --pre autogluon` and it still installed 0.7.0 (I don't know if I did sth wrong, or it is from `autogluon`). Anyway, I tried with AutoGluonTabular...
